basetest.pm

```diff
@@ -528,7 +534,12 @@ sub verify_sound_image {
         my $foundneedle = $rsp->{found};
         $self->record_screenmatch($img, $foundneedle, [$mustmatch], $rsp->{candidates});
         my $lastarea = $foundneedle->{area}->[-1];
-        bmwqemu::fctres(sprintf("found %s, similarity %.2f @ %d/%d", $foundneedle->{needle}->{name}, $lastarea->{similarity}, $lastarea->{x}, $lastarea->{y}));
+        bmwqemu::fctres(
+            sprintf(
+                "found %s, similarity %.2f @ %d/%d",
+                $foundneedle->{needle}->{name},
+                $lastarea->{similarity},
+                $lastarea->{x}, $lastarea->{y}));
         return $foundneedle;
```

Comment:
hmm, did you work on this manually or why is perltidy once putting )) on an extra line and once not?